src/styleguide/_data/data.json

```diff
@@ -32,14 +32,31 @@
     "city": "Visakhapatnam",
     "country": "India"
   },
-  "locations": [
-    {}, {}, {}
-  ],
+  "office": {
+    "name": "London",
+    "addressline1": "2 Finsbury Avenue",
+    "addressline2": "",
+    "city": "London",
+    "postcode": "EC2M 2PA",
+    "country": "UK",
+    "img": "http://via.placeholder.com/425x300/000000/ffffff",
+    "img-alt": ""
+  },
   "job": {
     "title": "Lead Implementation Planner",
     "type": "full-time"
   },
+  "image": {
+    "path": "http://via.placeholder.com/425x600/000000/ffffff",
+    "alt": ""
+  },
   "jobs": [
     {}, {}, {}
+  ],
+  "offices": [
+    {}, {}, {}, {}, {}, {}
```

Comment:
It would probably be better to have more, to see how it will adapt if we expand ❓

Reply:
Added a couple more